### ISSUE TITLE
No such thing as Northern Macedonia

### ISSUE THREAD

**Issue by egusheva** [calliope-project/euro-calliope#265]
### What happened?

It's best to change the region name to the official name 'North Macedonia'. 

### Version

1.2.0.dev

**Comment by timtroendle** [comment]
Fair point. As I see it, we are using the name only in two locations: config and locations.csv/.yaml. We may not be able to fix this easily in the config (because we are using dependencies that use the old name), but we should be able to change this easily in the locations.csv/.yaml. As the latter are the final product that's used and shipped (for example in the pre-built models), that seems fine to me. What do you think @egusheva?

**Comment by egusheva** [comment]
It seems fine to me as well. Thanks for the response @timtroendle

**Comment by timtroendle** [comment]
Actually, I figure I misunderstood. Euro-Calliope is using the old name "Macedonia", but it doesn't use the wrong name "Northern Macedonia". Where did you find this?
